### PR TITLE
Add EIP-6147: Guard of NFT/SBT, an Extension of EIP-721

### PR DESCRIPTION
**Pull request by 5660-eth**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-6147.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-12-07
+requires: 165, 721
+---
+
+## Abstract
+
+This standard is an extension of [EIP-721](./eip-721.md). It separates the holding right and transfer right of NFT/SBT and defines a new role, `guard`. The flexibility of the `guard` setting enables the design of NFT anti-theft, NFT lending, NFT leasing, SBT, etc.
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:16]: Until SBTs are more common, I'd recommend expanding the acronym:

```suggestion
This standard is an extension of [EIP-721](./eip-721.md). It separates the holding right and transfer right of non-fungible tokens (NFTs) and Soulbound Tokens (SBTs) and defines a new role, `guard`. The flexibility of the `guard` setting enables the design of NFT anti-theft, NFT lending, NFT leasing, SBT, etc.
```

File: EIPS/eip-6147.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+## Motivation
+
+NFT is an asset that has both use and financial value.
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:20]:
```suggestion
An NFT is an asset that has both use and financial value.
```

File: EIPS/eip-6147.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+
+Many cases of NFT theft currently exist, and current NFT anti-theft schemes, such as transferring NFT to cold wallets, make NFT inconvenient to be used.
+
+In current NFT lending, the NFT owner needs to transfer the NFT to the NFT lending contract, and the NFT owner no longer has the right to use the NFT while he or she has obtained the loan. In the real world, for example, if a person takes out a mortgage on his own house, he still has the right to use that house.
+
+For SBT. The current mainstream view is that SBT is not transferable, which makes SBT bound to an Ether address. However, when the private key of the user address is leaked or lost, retrieving SBT will become a complicated task and there is no corresponding specification. The SBTs essentially realizes the separation of NFT holding rights and transfer rights. When the wallet where SBT is located is stolen or unavailable, SBT should be able to be recoverable.
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:26]:
```suggestion
For SBT, the current mainstream view is that an SBT is not transferable, which makes a SBT bound to an Ether address. However, when the private key of the user address is leaked or lost, retrieving SBT will become a complicated task and there is no corresponding specification. The SBTs essentially realizes the separation of NFT holding rights and transfer rights. When the wallet where SBT is located is stolen or unavailable, SBT should be able to be recoverable. 
```

File: EIPS/eip-6147.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+
+
+## Specification
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### Contract Interface
+  
+```solidity
+ interface IERC721QS {
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:38]: This interface name is "alphabet soup", meaning it isn't very understandable.

File: EIPS/eip-6147.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+## Rationale 
+
+### Specification Management for Separation of Permissions
+
+The standard defines a new role `guard` and regulates the permissions of `owner` and `guard` as follows
+
+`owner`（Including authorized operators, which will not be described in detail below）: when the guard of the NFT is empty, `owner` can transfer the NFT, and also set `guard`. However, when `guard` already exists for the NFT, `owner` cannot modify `guard`, and cannot transfer the NFT.
+
+`guard`: The `guard` can remove its own `guard` identity or transfer the NFT to a specified address. For example, the `guard` can be set as the cold wallet address of the NFT holder, or an address trusted by the NFT holder. After the `owner` address of the NFT is abnormal, the `guard` can call the contract to transfer the NFT to the specified address.
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:89]: This should be in the specification section.

File: EIPS/eip-6147.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+
+### The design idea of the standard is as follows
+
+#### Universality
+
+There are many application scenarios for NFT/SBT, and there is no need to propose a dedicated eip for each specific application scenario, which would make the overall number of eips inevitably increase and add to the burden of developers. The standard is based on the analysis of the right attached to assets in the real world, and abstracts the right attached to NFT/SBT into holding right and transfer right making the standard more universal.
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:95]:
```suggestion
There are many application scenarios for NFT/SBT, and there is no need to propose a dedicated EIP for each one, which would make the overall number of EIPs inevitably increase and add to the burden of developers. The standard is based on the analysis of the right attached to assets in the real world, and abstracts the right attached to NFT/SBT into holding right and transfer right making the standard more universal.
```

File: EIPS/eip-6147.md [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+
+## Security Considerations
+
+When an NFT has a `guard`, even if an address is authorized as an operator through `approve` or `setApprovalForAll`, the operator still has no right to transfer the NFT.
+
+For NFT trading platforms that trade through `setApprovalForAll` + holder's signature, when NFT has `guard`, it cannot be traded. It is recommended to prevent such pending orders by checking the interface beforehand.
+
+## Copyright
+
+Copyright and related rights waived via CC0.
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:272]: Please use the exact text from the [EIP template](https://github.com/ethereum/EIPs/blob/master/eip-template.md#copyright)

**Comment by 5660-eth** [comment]
I'm sorry that my English is not so good that I made some mistakes in some English details. Thanks for suggestions for improvement.

File: EIPS/eip-6147.md [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+
+The `UpdateGuardLog` event MUST be emitted when a guard is changed.
+
+The `supportsInterface` method MUST return `true` when called with `0xc0655ef1`.
+
+## Rationale 
+
+### The design idea of the standard is as follows
+
+#### Universality
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:112]:
```suggestion
### Universality
```

File: EIPS/eip-6147.md [hunk 9]
```diff
@@ -0,0 +1,291 @@
...
+
+NFT anti-theft. If an NFT holder sets a `guard` address of an NFT as his or her own cold wallet address, the NFT can still be used by the NFT holder, but the risk of theft is greatly reduced.
+
+NFT lending. The borrower sets the `guard` of his or her own NFT as the lender's address, the borrower still has the right to use the NFT while obtaining the loan, but at the same time cannot transfer or sell the NFT. If the borrower defaults on the loan, the lender can transfer and sell the NFT.
+
+#### Simplicity
+
+Improvements to the ETH protocol should be as simple as possible. Entities should not be multiplied beyond necessity.
+
+#### Extensibility
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:128]:
```suggestion
### Extensibility
```

File: EIPS/eip-6147.md [hunk 10]
```diff
@@ -0,0 +1,291 @@
...
+  
+This standard only defines a `guard`, for the complex functions required by NFTs and SBTs, such as social recovery, multi-signature, expires management, according to the specific application scenarios, the `guard` can be set as a third-party protocol address, through the third-party protocol to achieve more flexible and diverse functions.
+
+### Choice of name
```
Comment (SamWilsn) [inline EIPS/eip-6147.md:132]:
```suggestion
### Naming
```

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**